lms/djangoapps/instructor/views/api.py

```diff
@@ -1405,7 +1405,7 @@ def _cohorts_csv_validator(file_storage, file_to_validate):
     Verifies that the expected columns are present in the CSV used to add users to cohorts.
     """
     with file_storage.open(file_to_validate) as f:
-        reader = unicodecsv.reader(UniversalNewlineIterator(f), encoding='utf-8')
```

Comment:
The UniversalNewlineIterator was handeling reading both `CR` and `CRLF` newline characters and trimming them.  Don't we need that still?  Or is the latest version of the unicodecsv module able to read `CR` and `CRLF` lines as equivalent?

Reply:
there was a problem with input to `UniversalNewlineIterator`, fixing that solved the issue, that I initially thought was an issue with `UniversalNewlineIterator`